### ISSUE TITLE
Man sollte nicht die ganzen Eingaben verlieren, nur weil ein Feld fehlt

### ISSUE THREAD

**Issue by bwl21** [mattimaier/bnote#336]
Wenn ein einem Formular ein Pflichtfeld fehlt, kommt eine rote Fehlermeldung, man muss zurückgehen und alles noch mal eintippen. Die Fehler sollten im Formular angezeigt werden, ggf. auch erst nach Klick auf dem speichern. Aber man sollte nicht die ganzen Eingaben verlieren.

Leider sind auch nicht alle Felder als Pflichtfelder gekennzeichnet ...

**Comment by mattimaier** [comment]
Das ist leider ein altes Problem. Teilweise konnte ich es schon verbessern. Allerdings unterstützen fast alle modernen Browser (z.B. Chrome, Safari und Firefox) die "Formular merken" Funktion, sodass du einfach auf "Zurück" im Browser klicken kannst und deine Eingaben vorhanden sind.
Kannst du mir ein paar Formulare nennen um konkreter die Themen angehen zu können?

**Comment by bwl21** [comment]
Ja, ich sehe, dass mit dem Browser-Zurück, die Felder noch da sind. Das hatte ich erst gar nicht probiert, weil es ja auch in der Applikation einen [Zurück] Button gibt. 

Z.B. bei einem Proberaum ist der name nicht als Pflichtfeld gekennzeichnet.

Wie gesagt, ideal wäre schon, wenn der Fehler beim Verlassen des Feldes angezeigt würde.
